assets/src/onboarding-wizard/pages/done/index.js

```diff
@@ -0,0 +1,194 @@
+import { NavMenu } from '../../../components/nav-menu';
+import { READER, STANDARD, TRANSITIONAL } from '../../../common/constants';
+import { Preview } from './preview';
+import { Saving } from './saving';
+import { usePreview } from './use-preview';
+
+/**
+ * Final screen, where data is saved.
+ */
+export function Done() {
+	const {
+		didSaveOptions,
+		editedOptions: { theme_support: themeSupport, reader_theme: readerTheme },
+		hasOptionsChanges,
+		readerModeWasOverridden,
+		saveOptions,
+		savingOptions,
+	} = useContext( Options );
+	const { didSaveDeveloperToolsOption, saveDeveloperToolsOption, savingDeveloperToolsOption } = useContext( User );
+	const { canGoForward, setCanGoForward } = useContext( Navigation );
+	const { downloadedTheme, downloadingTheme, downloadingThemeError } = useContext( ReaderThemes );
+	const { previewLinks, setActivePreviewLink, previewUrl, isPreviewingAMP, toggleIsPreviewingAMP } = usePreview();
+
+	/**
+	 * Allow the finish button to be enabled.
+	 */
+	useEffect(
+		() => {
+			if ( ! canGoForward ) {
+				setCanGoForward( true );
+			}
+		},
+		[ setCanGoForward, canGoForward ],
+	);
+
+	/**
+	 * Triggers saving of options on arrival to this screen.
+	 */
+	useEffect( () => {
+		if ( ! didSaveOptions && ! savingOptions ) {
+			saveOptions();
+		}
+	}, [ didSaveOptions, saveOptions, savingOptions ] );
+
+	/**
+	 * Triggers saving of user options on arrival of this screen.
+	 */
+	useEffect( () => {
+		if ( ! didSaveDeveloperToolsOption && ! savingDeveloperToolsOption ) {
+			saveDeveloperToolsOption();
+		}
+	}, [ didSaveDeveloperToolsOption, savingDeveloperToolsOption, saveDeveloperToolsOption ] );
+
+	if ( savingOptions || savingDeveloperToolsOption || downloadingTheme || hasOptionsChanges ) {
+		return <Saving />;
+	}
+
+	return (
+		<div className="done">
+			<h1 className="done__heading">
+				{ __( 'Done', 'amp' ) }
+			</h1>
+			<div className="done__content done__content--primary">
+				<h2 className="done__icon-title">
+					<IconLaptopSearch />
+					{ __( 'Review', 'amp' ) }
+				</h2>
+				{ READER === themeSupport && downloadedTheme === readerTheme && (
+					<AMPNotice size={ NOTICE_SIZE_LARGE } type={ NOTICE_TYPE_SUCCESS }>
+						{ __( 'Your Reader theme was automatically installed', 'amp' ) }
+					</AMPNotice>
+				) }
+				{ readerModeWasOverridden && (
+					<AMPNotice type={ NOTICE_TYPE_INFO } size={ NOTICE_SIZE_LARGE }>
+						{ __( 'Because you selected a Reader theme that is the same as your site\'s active theme, your site has automatically been switched to Transitional template mode.', 'amp' ) }
+					</AMPNotice>
+				) }
+				<p>
+					{ __( 'Your site is ready to bring great experiences to your users!', 'amp' ) }
+				</p>
+				{ STANDARD === themeSupport && (
+					<p>
+						{ __( 'In Standard mode there is a single AMP version of your site. Browse your site below to ensure it meets your expectations.', 'amp' ) }
```

Comment:
Since on desktop the preview is to the side and not below.

```suggestion
						{ __( 'In Standard mode there is a single AMP version of your site. Browse your site here to ensure it meets your expectations.', 'amp' ) }
```